lib/view_component/test_helpers.rb

```diff
@@ -190,6 +190,29 @@ def with_request_url(path, host: nil)
       @vc_test_controller = old_controller
     end
 
+    # Set the controller helpers to be used while executing the given block,
+    #
+    # ```ruby
+    # with_helpers(UsersHelper) do
+    #   render_inline(MyComponent.new)
+    # end
+    # ```
+    #
+    # @param *args [Array<Module>] The helpers to be used.
+    def with_helpers(*args)
+      args.each do |helper_mthd|
```

Comment:
```suggestion
      args.each do |helper_method|
```

Can we avoid shortening variable names?

Reply:
moving this to a new pr